nutanix/resource_nutanix_nbd_database_test.go

```diff
@@ -29,6 +29,25 @@ func TestAccEra_basic(t *testing.T) {
 	})
 }
 
+func TestAccEraDatabaseProvisionHA(t *testing.T) {
+	name := "test-pg-inst-HA-tf"
+	desc := "this is desc"
+	sshKey := testVars.SSHKey
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccEraPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccEraDatabaseHAConfig(name, desc, sshKey),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
Can we assert few more important fields like DB servers, profiles, etc. to avoid regressions ?